src/engraving/libmscore/score.h

```diff
@@ -1233,8 +1229,9 @@ class Score : public EngravingObject
     virtual String name() const;
 
     void cmdTimeDelete();
-    void localTimeDelete();
-    void globalTimeDelete();
+    bool checkTimeDelete(Segment* startSegment, Segment* endSegment);
+    void doTimeDelete(Segment* startSegment, Segment* endSegment);
+    void doDoTimeDelete(Measure*, Segment*, const Fraction&);
```

Comment:
**doDo**TimeDelete? :) This name looks weird :)

Reply:
Yeah, I was a bit inspirationless 😅 Changed it to something a bit more meaningful now.